### PR TITLE
Done registerPage with additional information: phone, name, dateOfBirth"

### PR DESCRIPTION
**Pull request by tracytrle**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import NotFoundPage from "./pages/Register/NotFoundPage";
 import RegisterPage from "./pages/Register/RegisterPage";
 import { AuthProvider } from "./context/authContext/AuthContext";
 import InfoForm from "./pages/Register/InfoForm";
+
 // import ProtectedRoute from "./routes/ProtectedRoute";
```
Comment (travistrle) [inline src/App.js]: Can you delete this line next PR?

File: src/api/api.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import axios from "axios";
 import { API_URL } from "./config";
 
-export const register = (email, password) => {
-  return axios.post(`${API_URL}/auth/register`, { email, password });
+// export const register = (email, phone, password) => {
```
Comment (travistrle) [inline src/api/api.js]: Please remove the code if you don't use it. Comment code is a bad habit. We can always use git to go back to previous change

File: src/components/Forms/ClientInfo.js [hunk 3]
```diff
@@ -10,8 +11,75 @@ import MedicalHistory from "./MedicalHistory";
 import PhoneInput from "./PhoneInput";
 import { useTranslation } from "react-i18next";
 
+const genders = ["Male", "Female", "Not Specific"];
+
 export default function FormPropsTextFields() {
   const { t } = useTranslation();
+  const [firstName, setFirstName] = useState("");
```
Comment (travistrle) [inline src/components/Forms/ClientInfo.js]: This works! But I think a better way to implement this is to define a DTO class called `User`. 
So you will only need one useState(defaultUser).

File: src/components/Forms/ClientInfo.js [hunk 4]
```diff
@@ -10,8 +11,75 @@ import MedicalHistory from "./MedicalHistory";
...
+      email,
+      phone,
+      address,
+      city,
+      country,
+      birthDate,
+    };
+  };
+  React.useEffect(() => {
+    console.log("firstName", firstName);
```
Comment (travistrle) [inline src/components/Forms/ClientInfo.js]: please remove the console. Production code shouldn't have console

File: src/components/Forms/DatePickerComp.js [hunk 5]
```diff
@@ -1,21 +1,28 @@
...
 import { DemoContainer } from "@mui/x-date-pickers/internals/demo";
 import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import { DatePicker } from "@mui/x-date-pickers/DatePicker";
 import { useTranslation } from "react-i18next";
 import Box from "@mui/material/Box";
+import { use } from "i18next";
 
-export default function DatePickerValue() {
+export default function DatePickerComp({ callback }) {
```
Comment (travistrle) [inline src/components/Forms/DatePickerComp.js]: I suggest name `callback` as `callbackHandler`

Comment (travistrle) [inline src/components/Forms/DatePickerComp.js]: DatePickerComp is inside a component? I don't think you need suffix `comp`

File: src/App.js (hunk 1 above)
Comment (tracytrle) [reply src/App.js]: Done

File: src/api/api.js (hunk 2 above)
Comment (tracytrle) [reply src/api/api.js]: Done

File: src/pages/Register/RegisterPage.js [hunk 6]
```diff
@@ -7,28 +7,90 @@ import { useNavigate } from "react-router-dom";
...
+import { IconButton } from "@mui/material";
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+const genders = ["Male", "Female", "NS"];
 
 function RegisterPage() {
   const { t } = useTranslation();
   const theme = useTheme();
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [phone, setPhone] = useState("");
```
Comment (travistrle) [inline src/pages/Register/RegisterPage.js:24]: same as comment above. I think use DTO user class is a better approach

File: src/components/Forms/ClientInfo.js (hunk 4 above)
Comment (tracytrle) [reply src/components/Forms/ClientInfo.js]: Will work on it later

File: src/components/Forms/ClientInfo.js (hunk 3 above)
Comment (tracytrle) [reply src/components/Forms/ClientInfo.js]: That is a good idea! I plan to refactor the code later

File: src/components/Forms/DatePickerComp.js (hunk 5 above)
Comment (tracytrle) [reply src/components/Forms/DatePickerComp.js]: OK

**Review by travistrle — CHANGES_REQUESTED** [review]
I'd suggest you update your PR.

**Review by travistrle — APPROVED** [review]
LGTM. Those comments are minor, we can address it in the next PR.

**Pull request merged**